docs/api.md

```diff
@@ -182,6 +182,8 @@
   * [request.url](#requesturl)
 - [class: Response](#class-response)
   * [response.buffer()](#responsebuffer)
+  * [response.fromDiskCache](#responsefromdiskcache)
```

Comment:
If I understand it correctly, every response could be served either from network, disk cache or service worker. So it's never the case that both `fromDiskCache` and `fromServiceWorker` are `true`. Is this the case? 

If so, I'd rather do a `response.servedFrom()` method that returns either `network`, `diskcache` or `serviceworker`.

Reply:
They can both be true at the same time. If you check a request against `https://googlechrome.github.io/samples/service-worker/basic/`, it shows both `fromDiskCache` and `fromServiceWorker` as true. From what I understand `fromServiceWorker` is when the request is served from the `fetch` event inside the web worker. But fromDiskCache will also return true if you return from the Cache API.

Hope I haven't misunderstood your question.

Reply:
It looks like they're different....running against chromestatus shows all 4 permutations can happen:

<img width="733" alt="screen shot 2017-12-16 at 8 43 22 am" src="https://user-images.githubusercontent.com/238208/34072496-466c36f4-e23d-11e7-829f-c0c5f8b7eec3.png">

That makes sense because the disk cache is different than the sw cache storage api. Pretty nice that DTs has that level of info.